framework/vendors/markdown/markdown.php

```diff
@@ -1455,9 +1455,9 @@ public function _initDetab() {
 	# regular expression.
 	#
 		if (function_exists($this->utf8_strlen)) return;
-		$this->utf8_strlen = create_function('$text', 'return preg_match_all(
-			"/[\\\\x00-\\\\xBF]|[\\\\xC0-\\\\xFF][\\\\x80-\\\\xBF]*/",
-			$text, $m);');
+		$this->utf8_strlen = function ($text) use ($m) {
```

Comment:
Can't be done like that because `"php": ">=5.1.0"`. `create_function` should be left as fallback.

Reply:
I understand, but what are the chances of people still using php 5.1 though? ;-)

Reply:
[About 7/100](https://w3techs.com/technologies/details/pl-php/5/all).